src/api/endpoints/index.ts

```diff
@@ -1,7 +1,50 @@
-export const AUTH_GW_ENDPOINT =
-  'https://q88uo5prmh.execute-api.eu-north-1.amazonaws.com';
+export const APP_BASE_URL = (() => {
```

Comment:
Nämä urlit voisi tosiaan laittaa tulemaan sieltä pulumista tai github-actionsseista environment-muuttujaksi niin ei tarvitsisi koodissa säilöä | tehdä stage-iffailuja. Eli että olisikin vain jotain tyyliin: `export const APP_BASE_URL = process.env.APP_BASE_URL` 

Nice to have!